src/nfs.c

```diff
@@ -295,6 +301,19 @@ static kstat_t *nfs4_ksp_client;
 static kstat_t *nfs4_ksp_server;
 #endif
 
+static int nfs_config(const char *key, const char *value) {
+  if (strcasecmp(key, "IgnoreV2") == 0 && IS_TRUE(value))
+    ignore_v2 = 1;
```

Comment:
Please just assign the valur of `IS_TRUE()`, i.e. `ignore_v2 = IS_TRUE(value)`. This allows (convoluted but correct) configs such as

```
<Plugin nfs>
  IgnoreV2 true
  IgnoreV2 false
</Plugin>
```